interview-questions.md

```diff
@@ -8,50 +8,51 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: Parameters help us to define what type of data can be passed and acted upon based on out argument. Parameters can only accept one value where as arguments are dynamic and can pass multiple arguments and parameters.
 
-Researched answer:
+Researched answer: Functions need something to run and it takes in parameters like type, index, or array. The argument is what takes those parameters and performs some kind of logic.
```

Comment:
I think I understand what you are saying, just need some clarification. Parameters are place holders while arguments are values. Parameters allow you to dynamically accept any values as arguments within the function call aka function invocation aka invoking the function.